dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -16,23 +16,11 @@
 
 package io.dockstore.webservice.helpers;
 
-import java.io.IOException;
```

Comment:
Large import rearrangements, might be building with wrong Maven version or Java version locally?
We're using https://code.revelc.net/impsort-maven-plugin/ so its a bit odd

Reply:
Hm, I used `./mvnw clean install` to build

Reply:
the changes look correct to me. seems like something's wrong with the previous